lib/gooddata/extensions/array.rb

```diff
@@ -0,0 +1,18 @@
+class Array
+  # Converts an array into a string suitable for use as a URL query string, using the given key as the param name.
+  def to_query(key)
+    prefix = "#{key}[]"
+
+    if empty?
+      nil.to_query(prefix)
+    else
+      collect { |value| value.to_query(prefix) }.join '&'
+    end
+  end
+
+  # Calls to_param on all its elements and joins the result with slashes.
+  # This is used by url_for in Action Pack.
+  def to_param
+    collect(&:to_param).join('/')
+  end
+end
```

Comment:
Can we make both static helpers? It does not seem there is any additional benefit to have them directly on the Array and since come customers are using this as a lib we should be as unintrusive as possible.